packages/react-dom/src/client/ReactDOMRoot.js

```diff
@@ -173,13 +186,16 @@ export function createRoot(
     isStrictMode,
     concurrentUpdatesByDefaultOverride,
     identifierPrefix,
+    transitionCallbacks,
   );
   markContainerAsRoot(root.current, container);
 
   const rootContainerElement =
     container.nodeType === COMMENT_NODE ? container.parentNode : container;
   listenToAllSupportedEvents(rootContainerElement);
 
+  ReactCurrentHostConfig.current = ReactDOMCurrentHostConfig;
```

Comment:
Can you remind me why this is being done? Is the idea that you might have nested renderers of different types, and you want to key their transitions differently?